packages/fixie/src/agent.ts

```diff
@@ -500,7 +500,7 @@ export class FixieAgent {
     environmentVariables: Record<string, string> = {}
   ): Promise<AgentRevision> {
     const config = await FixieAgent.LoadConfig(agentPath);
-    const agentId = `${(await client.userInfo()).username}/${config.handle}`;
+    const agentId = config.handle;
```

Comment:
This is a misleading variable name -- can you remove the variable and just use `config.handle` everywhere?